monkey/infection_monkey/master/exploiter.py

```diff
@@ -53,6 +53,7 @@ def exploit_hosts(
         exploitation_config: ExploitationConfiguration,
         hosts_to_exploit: Queue,
         current_depth: int,
+        servers: Sequence[str],
```

Comment:
Maybe it's worth moving the servers to be object property? Most of Exploiter methods seem to use this parameter

Reply:
Perhaps not, but these method definitions are growing larger and larger. This class pass around a bunch of parameters it doesn't use and it doesn't need to know depth, servers or targets. It needs to know  only stop event and victim OS by the looks of it

Reply:
In the future, the list of severs may be updated with the list of all known servers from the island. The exploiter needs to be able to receive an updated list every time it runs.